src/DOM/HTML/Window.js

```diff
@@ -201,3 +201,33 @@ exports.sessionStorage = function (window) {
     return window.sessionStorage;
   };
 };
+
+exports._requestAnimationFrame = function(fn) {
+  return function(window) {
+    return function() {
+      return window.requestAnimationFrame(fn);
+    };
+  };
+};
+
+exports._cancelAnimationFrame = function(id) {
+  return function(window) {
+    return window.cancelAnimationFrame(id);
```

Comment:
Need an extra `function() {` wrapping the inner thing here as we're `Eff`-typing these things.